.travis.yml

```diff
@@ -12,9 +12,14 @@ sudo: required
 cache:
   directories:
     - $GOPATH/src/gx
+    - $GOPATH/pkg/mod
+    - $HOME/.cache/go-build
 
 install: true
 
+env:
```

Comment:
This should follow the [gomod/ci-example](https://github.com/ipfs/ci-helpers/blob/master/travis-ci/travis.example.yml)

Reply:
no, we have a proper multi-stage build. Plus not making cluster CI depend on a complex external script that can change under our feet. ok for small libraries.

Reply:
@hsanjuan actually forget about using the example, but there ci-helpers script does have the functionality of untangling gx from go-mod so that things run in the proper order and things are rewritten when they shouldn't be.